internal/k8s/client.go

```diff
@@ -192,9 +192,9 @@ func (w *ClientWrapper) CoreDNSMatch() (bool, error) {
 			continue
 		}
 
-		split := strings.Split(c.Image, ":")
-		if len(split) == 2 {
-			version = split[1]
+		split := strings.LastIndex(c.Image, ":")
+		if split != -1 {
+			version = c.Image[split + 1:]
```

Comment:
Unfortunately, if the last character of the image field is a `:` this will panic.
WDYT of keeping the `strings.Split()` but with picking the last item of the split instead ? (Something like [this](https://play.golang.org/p/0ERZcKdyqVm)).